app/workers/subject_set_statuses_create_worker.rb

```diff
@@ -16,8 +16,10 @@ def perform(subject_set_id, workflow_id)
       .where(subject_set_id: subject_set_id)
       .select(:id,:subject_id)
 
+    duration = linked_subject_select_scope.length * 4
```

Comment:
use count here to count in sql, if you use length here it'll load all the object to RAM to count them. That can be very bad.

Reply:
dang. nice catch, man. always learnin' over here.

Reply:
i've made that mistake too many times ;)